### PR TITLE
✨amp-experiment-v1: Created experiment for amp-experiment-v1

### PR DESCRIPTION
**Pull request by torch2424**
Ironic title 😂 

This creates the experiment check for `amp-experiment-v1` with support for origin trial checking (as I trust we will be going down that path, cc @jeffjose ).

# Examples

**Before Enabling Experiment**

Screen Shot 2019-04-01 at 3 02 18 PM

**After Enabling Experiment**

Screen Shot 2019-04-01 at 3 02 55 PM

### CODE REVIEW

File: extensions/amp-experiment/1.0/amp-experiment.js [hunk 1]
```diff
@@ -34,31 +39,50 @@ export class AmpExperiment extends AMP.BaseElement {
...
+      const variantsService = responses[0];
+      const enabled = responses[1];
+
+      if (!enabled) {
+        user().error(TAG, 'Experiment amp-experiment-1.0 is not enabled.');
+
+        // Ensure downstream consumers don't wait for the promise forever.
+        variantsService.init(Promise.resolve({}));
+
+        return Promise.reject(
```
Comment (zhouyx) [inline extensions/amp-experiment/1.0/amp-experiment.js]: Is `Promise.reject()` handled well by `buildCallback()`. It won't lead to uncaught dev error right

Comment (torch2424) [reply extensions/amp-experiment/1.0/amp-experiment.js]: In the screenshots, the only ugly error I saw was "Uncaught Error: _reported_ ...". Which I found to also happen for other experiments (reCAPTCHA). Which I think is a recent thing in master, as it originally did not do that.

That being said, this code is essentially copy-paste from the reCAPTCHA experiment, so I assume it does not? 🤔 

I'll wait for your reply before merging @zhouyx 😄

Comment (zhouyx) [reply extensions/amp-experiment/1.0/amp-experiment.js]: I looked at the source code. 
https://github.com/ampproject/amphtml/blob/47bd44948d0c57bd3c8e8b9cbb19b92c1e7c3b7a/src/custom-element.js#L536-L542

We reportError on this, which I believe should not be reported. But that's not blocking, we can fix it later.

Comment (torch2424) [reply extensions/amp-experiment/1.0/amp-experiment.js]: Want me to create an issue for this? 😄 But glad this is not blocking the PR 👍

**Comment by torch2424** [comment]
@zhouyx 's comment is unrelated to this PR. Thus, this is approved. Merging... 😄 🎉 

Thanks for the review @zhouyx !

**Pull request merged**